.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+<!-- Add issue number here. If you do not solve the issue entirely, please change the message e.g. "Addresses #IssueNumber -->
+Fixes #
+
+#### Checklist
```

Comment:
Let's make it personal checklist, people don't really need to include that. We don't want contributors to be put off by a very rigid template.

Reply:
So to remove content of the checklist or its checkboxes as well ?

Reply:
Just name it 'Personal Checklist'.